settings.gradle

```diff
@@ -102,7 +102,6 @@ include "servicetalk-annotations",
         "servicetalk-http-security-jersey3-jakarta10",
         "servicetalk-http-utils",
         "servicetalk-loadbalancer",
-        "servicetalk-loadbalancer-experimental",
```

Comment:
From offline discussion: consider keeping this as an empty jar for the next 1-2 releases to make sure we don't break anyone's build accidentally.